python/kvikio/tests/test_cufile_driver.py

```diff
@@ -6,6 +6,10 @@
 import kvikio.cufile_driver
 
 
+def test_version():
+    print(kvikio.cufile_driver.libcufile_version())
```

Comment:
Please assert some property here, like that it is an tuple of two elements with each value >=0. Avoid printing in tests.

Reply:
done